sources/osg/Matrix.js

```diff
@@ -1223,6 +1223,46 @@ define( [
             };
         } )(),
 
+        multScale: function ( x, y, z, m, res ) {
+
+            if ( !res ) {
```

Comment:
k, so we rename it preMultScale for consistency and we'll make the multScale with a matrix result != m .